src/Features/Core/Portable/GenerateConstructorFromMembers/GenerateConstructorFromMembersCodeRefactoringProvider.cs

```diff
@@ -129,7 +129,7 @@ private async Task HandleNonSelectionAsync(CodeRefactoringContext context)
                 new GenerateConstructorWithDialogCodeAction(
                     this, document, textSpan, containingType, viableMembers,
                     pickMemberOptions.ToImmutableAndFree()),
-                textSpan);
+                typeDeclaration.Span);
```

Comment:
that's a big span... intentional?  what's the experience like here?  does this effectively mean this refactoring shows up everywhere in this class?  Are you sure we want to move to this?

Reply:
No, that's just ordering thing. The generateXXX refactorings have applicableToSpan the whole type which should - in theory - make them furthest from current caret.

That said, in situations where they show up (on header & in between types) there are unlikely to be any other refactorings so it's essentially no-change.

Reply:
oh right.  definitely confused me for a moment.  i wonder if the applicable-to-span should just be the start of the type decl.  but i suppose it's not really going to make much of a difference either way.  So this is fine by me :)